gen3/lib/utils.sh

```diff
@@ -362,3 +362,29 @@ gen3_encode_uri_component() {
     echo "$str"
   )
 }
+
+
+#
+# if the module has a manifest, most likely there is a terraform version
+# value that would help us determine which terraform version to use
+#
+checkTerraformModule() {
```

Comment:
Why do we have 2 of these?

Reply:
I think I just renamed the one in `gen3/bin/workon.sh` before removing it because I thought it was the place for the function to be, but at some point, it was not the case so I had to move it to `gen3/lib/utils.sh`

I'll remove the duplication.